django_short_urls/tests/models_link_test.py

```diff
@@ -135,3 +136,11 @@ def test_prefix(self):
         prefix = 'foo'
         link, _ = _shorten(self.URL, prefix=prefix)
         self.assertEqual(link.prefix, prefix)
+
+    def test_build_absolute_uri(self):
+        link, _ = _shorten("http://www.work4labs.com/", short_path='foo')
+
+        self.assertEqual(
+            link.build_absolute_uri(RequestFactory().get('/test')),
+            "https://testserver/foo"
```

Comment:
I'm so puzzled about it being the first test actually checking the full URI − means there is no regression test here.